doc/book/box/stream.rst

```diff
@@ -45,6 +45,9 @@ The ID is generated on the client side automatically.
 If a user writes their own connector and wants to use streams,
 they must transmit the ``stream_id`` over the iproto protocol.
 
+Streams only work if the ``box.cfg{}`` option :ref:`memtx_use_mvcc_engine <cfg-basic-memtx-use-mvcc-engine>`
+is enabled on the server: ``memtx_use_mvcc_engine = true``.
```

Comment:
This is not true, streams have two tasks:
- to ensure the order of execution of requests within the stream
- interactive transactions.
Transactions will not work for memtx engine, but the order of execution of requests will be provided. In addition, no one bothers to create a stream and use it just like a connection. You can write that interactive transactions will not work for memtx engine without this option.